public/app/features/plugins/admin/components/PluginListCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { Icon, useStyles2, CardContainer, VerticalGroup } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { CatalogPlugin } from '../types';
+import { PluginLogo } from './PluginLogo';
+import { PluginBadges } from './PluginBadges';
+
+const LOGO_SIZE = '48px';
+
+enum IconName {
+  app = 'apps',
+  datasource = 'database',
+  panel = 'credit-card',
+  renderer = 'pen',
+}
+
+export function PluginListCard({ plugin }: { plugin: CatalogPlugin }) {
+  const { name, id, orgName, type } = plugin;
+  const styles = useStyles2(getStyles);
+
+  return (
+    <CardContainer href={`/plugins/${id}`} className={styles.cardContainer}>
+      <VerticalGroup spacing="md">
+        <div className={styles.headerWrap}>
+          <PluginLogo
+            src={plugin.info.logos.small}
+            alt={`${plugin.name} logo`}
+            className={styles.image}
+            height={LOGO_SIZE}
+          />
+          <h3 className={styles.name}>{name}</h3>
+          {type && (
+            <div className={styles.icon}>
+              <Icon name={IconName[type]} aria-label={`${type} plugin icon`} />
```

Comment:
Nit: Really great you added the aria-label 🙌🏾 , the only thing related to UX that probably we should address is to map the icon of each plugin with the type

Before (we had the text that described which type was this plugin)

![BeforePluginsCatalog](https://user-images.githubusercontent.com/239999/128163532-3daf8707-ca1a-4f13-863d-c083b17ca1da.png)


Now (We have a new icon, I feel is missing a tooltip|title with the type?)

![currentPluginsCatalog](https://user-images.githubusercontent.com/239999/128163554-f5cca31d-e686-4ea4-8c3c-d23542dd2048.png)